steps/pac/pac.go

```diff
@@ -0,0 +1,33 @@
+package pac
+
+import (
+	"github.com/getgauge-contrib/gauge-go/gauge"
+	"github.com/openshift-pipelines/release-tests/pkg/k8s"
+	"github.com/openshift-pipelines/release-tests/pkg/pac"
+	"github.com/openshift-pipelines/release-tests/pkg/pipelines"
+	"github.com/openshift-pipelines/release-tests/pkg/store"
+)
+
+var _ = gauge.Step("Configure Gitlab token for PAC tests", func() {
+
+	pac.ConfigureGitlabToken()
+})
+
+var _ = gauge.Step("Create Smee Deployment with <elname>", func(elname string) {
+
+	pac.SmeeDeployment(elname)
+	k8s.ValidateDeployments(store.Clients(), store.Namespace(), store.GetScenarioData("smee_deployment_name"))
+
+})
+
+var _ = gauge.Step("Configure & Validate Gitlab repo for pipelinerun", func() {
```

Comment:
The code is breaking here due to a recent code change. Will fix it in next commit